CMakeLists.txt

```diff
@@ -330,6 +334,7 @@ if (NOT WIN32)
 endif ()
 
 add_library(freeorioncommon
+    $<$<CXX_COMPILER_ID:msvc>:STATIC>
```

Comment:
I think it possible to change to shared libraries but now FO_COMMON_API isn't defined for windows.